cypress/e2e/scenario-1-environment.cy.js

```diff
@@ -112,13 +112,9 @@ describe("Environment", () => {
     cy.get("button").contains("Submit").should("be.disabled");
     cy.get('[aria-label="Name-input"]').type(testName(2));
     cy.get("button").contains("Submit").click();
-    cy.wait("@createEnv", { timeout: 10000 })
-      .its("response.statusCode")
-      .should("eq", 200);
-    cy.wait("@getCatalog");
     //go back to gome and check if env is visible
-
-    cy.get(".pf-c-breadcrumb__item").contains("Home").click();
+    cy.wait(500);
```

Comment:
`cy.get('[aria-label="Environment card"]', { timeout: 20000))....` The wait can end up being flakey I fear on Jenkins

Reply:
unfortunately it happened to be the other way round,I don't know why, but cy.wait resolved my issues and I didn't have any problems with flakiness so far